edu-1488-chatgpt-plugin-guide

MarianaAguilera opened the pull request:
New guide to explain how to use the template.

MarianaAguilera committed:
edu-1488-chatgpt-plugin-guide

New guide to explain how to use the template.

bru-andrade reviewed (commented):
Alguns comentários!

MarianaAguilera committed:
Apply suggestions from code review

Co-authored-by: Bru Andrade <119969414+bru-andrade@users.noreply.github.com>
Co-authored-by: hannah <109289700+hannah-sk@users.noreply.github.com>

MarianaAguilera committed:
Apply suggestions from code review

Co-authored-by: hannah <109289700+hannah-sk@users.noreply.github.com>

MarianaAguilera committed:
improvements

Link to marketplace.
Key configurations, writing more aligned with guides.

MarianaAguilera committed:
Apply suggestions from code review

Co-authored-by: hannah <109289700+hannah-sk@users.noreply.github.com>

MarianaAguilera committed:
Update src/content/docs/en/pages/guides/marketplace/chatgpt-plugin/chatggpt-plugin.mdx

Co-authored-by: hannah <109289700+hannah-sk@users.noreply.github.com>

hannah-sk reviewed (approved):
deixei mais algumas correçõezinhas menores!

MarianaAguilera committed:
Update src/content/docs/en/pages/guides/marketplace/chatgpt-plugin/chatggpt-plugin.mdx

Co-authored-by: hannah <109289700+hannah-sk@users.noreply.github.com>

MarianaAguilera committed:
Update src/content/docs/pt-br/pages/guias/marketplace/chatgpt-plugin/chatgpt-plugin.mdx

Co-authored-by: hannah <109289700+hannah-sk@users.noreply.github.com>

MarianaAguilera committed:
Apply suggestions from code review

Co-authored-by: hannah <109289700+hannah-sk@users.noreply.github.com>

MarianaAguilera committed:
Update src/content/docs/en/pages/guides/marketplace/chatgpt-plugin/chatggpt-plugin.mdx

MarianaAguilera committed:
Update src/content/docs/pt-br/pages/guias/marketplace/chatgpt-plugin/chatgpt-plugin.mdx

MarianaAguilera committed:
Update src/content/docs/en/pages/guides/marketplace/chatgpt-plugin/chatggpt-plugin.mdx

MarianaAguilera committed:
Apply suggestions from code review

MarianaAguilera committed:
improvements

-file name.
-template naming.
-devdetails.
-github token info.
-continuous deployment info.

MarianaAguilera committed:
no marketplace

MarianaAguilera committed:
Apply suggestions from code review

MarianaAguilera committed:
Apply suggestions from code review

MarianaAguilera committed:
Update src/content/docs/en/pages/guides/marketplace/chatgpt-plugin/chatgpt-plugin.mdx

MarianaAguilera committed:
Update src/content/docs/pt-br/pages/guias/marketplace/chatgpt-plugin/chatgpt-plugin.mdx

MarianaAguilera committed:
Improvements

GitHub tip.
Continuous deployment fix.
